apps/datahub/src/app/app.module.ts

```diff
@@ -229,6 +238,30 @@ export const metaReducers: MetaReducer[] = !environment.production ? [] : []
       provide: ORGANIZATION_URL_TOKEN,
       useValue: `${ROUTER_ROUTE_SEARCH}?${ROUTE_PARAMS.PUBLISHER}=\${name}`,
     },
+    {
+      provide: DO_NOT_USE_DEFAULT_BASEMAP,
+      useFactory: () => getOptionalMapConfig()?.DO_NOT_USE_DEFAULT_BASEMAP,
+    },
+    {
+      provide: BASEMAP_LAYERS,
+      useFactory: () =>
+        getOptionalMapConfig()?.MAP_LAYERS.map(getMapContextLayerFromConfig),
```

Comment:
I might be missing something, but shouldn't the token rather be called `MAP_LAYERS`?

Reply:
The token is used to add "base layers", layers that will be shown below any other layer that we ask for; I think the name "basemap layers" make sense. It is called "map layers" in the config but that name would be more confusing I think.

Reply:
Ah indeed, I ignored the fact that the config layers are base layers.